examples/bigquery/main.tf

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  credentials = "${file(var.credentials_path)}"
+}
+
+# Example of a org-level sink to a BigQuery dataset
+module "org_sink" {
+  source = "../../"
+  name   = "test-org-sink-bigquery"
+  org_id = "${var.org_id}"
+  filter = "severity > WARNING"
+
+  bigquery = {
+    name    = "org_sink"
+    project = "${var.project_name}"
+  }
+}
+
+# Example of a folder-level sink to a BigQuery dataset
+data "google_active_folder" "folder" {
+  display_name = "${var.folder_name}"
+  parent       = "organizations/${var.org_id}"
```

Comment:
Minor nit but this assumes we're exporting a root-level folder (fine to keep it though, since it's just an example).

Reply:
How about adding the following below the comment ?

```
# Note that this uses a root-level folder, but can be applied with any nested
# folder. 
```

Reply:
Sure.